apps/dashboard/app/(app)/apis/[apiId]/keys/[keyAuthId]/new/client.tsx

```diff
@@ -634,6 +649,61 @@ export const CreateKey: React.FC<Props> = ({ apiId, keyAuthId }) => {
                                   </FormItem>
                                 )}
                               />
+                             <FormField
+                                control={form.control}
+                                name="limit.refill.interval"
+                                render={({ field }) => (
+                                  <FormItem className="">
+                                    <FormLabel>Refill Rate</FormLabel>
+                                    <Select
+                                      onValueChange={field.onChange}
+                                      defaultValue="none"
+                                      value={field.value}
+                                    >
+                                      <SelectTrigger>
+                                        <SelectValue />
+                                      </SelectTrigger>
+                                      <SelectContent>
+                                        <SelectItem value="none">None</SelectItem>
```

Comment:
you removed "none" from the possible values above I think

Reply:
Correct. missed this one I guess.